cms/project_template/project_name/static/js/src/main.js

```diff
@@ -0,0 +1,3 @@
+import {hey} from './components/hello.js';
```

Comment:
Yes, just to show how the premise works.

I don't like putting examples in for the same reason we stripped stuff out before. But figured this would help clean up what is actually happening to people who haven't used modules before.